interview-questions.md

```diff
@@ -7,47 +7,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: A function is an operation that you would like to run. You would use a function to express logic.
 
+  Researched answer: A function is a set of instructions that specifically detail how to do a task. Functions make code reusable and can only be executed if it is called.
 
 
 2. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: An argument is a single point of logic while a parameter is the entire logic.
 
-  Researched answer:
+  Researched answer: Arguments are specified using parentheses. Additionally, arguments are the values that are passed through the function. On the other hand, parameters are initialized to the values of the arguments.  
```

Comment:
Yes! This difference is nuanced and a bit much to grasp at first but very useful!